docs/_templates/autosummary/module.rst

```diff
@@ -0,0 +1,14 @@
+{% extends "!autosummary/module.rst" %}
+
+.. Distributed under the MIT License.
+   See LICENSE.txt for details.
+
+.. Include pybindings documentation. This won't do anything for modules that
+   have no pybindings but print a warning.
```

Comment:
There's a lot of warnings. I get them locally and they are on CI. Is there a way to silence these?

Reply:
Can't silence these warnings because there's a hard-coded print in the sphinx code: https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/autosummary/generate.py#L535

Lesson: always use `logging`

Reply:
Rip...Guess we just need more pybindings then.

Reply:
I could remove _all_ output, but that's probably not a good solution either

Reply:
Yeah no we should keep the output